build.gradle.kts

```diff
@@ -67,9 +75,23 @@ tasks.named("assemble").configure {
 tasks.withType<JavaExec>().all {
 	jvmArgs("--enable-preview")
 	// Override the library path for tasks like run and test so they can find the libraries.
-	systemProperty("stacker.library.path", file("src/main/dist/native").absolutePath)
+	systemProperty("stacker.library.path", file("native").absolutePath)
 }
 
 tasks.withType<JavaCompile>().all {
 	options.compilerArgs!! += "--enable-preview"
 }
+
+val requireNativeTask by tasks.register("requireNativeDirectory") {
+	inputs.dir("native")
```

Comment:
Hmm, I can't do this. It yells at me when the file doesn't exist.

Reply:
You don't need incremental. It's fast. Do upToDate { false } so it always runs.